### DISCUSSION TITLE
traffice oppsite

### DISCUSSION THREAD

**Discussion by qoo7972365** [Security-Onion-Solutions/securityonion#10005] [General]
Hello I'm facing prolblem the traffice is oppsite,and I check log in forwarded node log "/nsm/zeek/logs/current"

![image](https://user-images.githubusercontent.com/32336596/227701636-24cb0be9-344f-4ba5-b749-94ded466d7d2.png)

and fellow uid check other log  I see conn.log also have wrong log,
![image](https://user-images.githubusercontent.com/32336596/227701706-7c3b0858-c26b-4e2a-bc08-9ce357dbe4d1.png)

My problem is  traffice should 
10.21.11.132:xxxxx(high port)  >>>  10.74.64.4 7443  this is correct
but always happen 
10.74.64.4:7443   >>>> 10.21.11.132:xxxxx(high port)  I think this is oppsite traffice

**Comment by InfosecGoon** [comment]
You should be getting both of those logs, representing the two sides of the network flow (from the client to the server, and the response from the server back to the client).

**Reply by qoo7972365** [reply]
This is security onion traffic log
![image](https://user-images.githubusercontent.com/32336596/228469446-028a819d-9bfd-41fc-a26e-657481193b54.png)

This is our fortinet traffic log
![image](https://user-images.githubusercontent.com/32336596/228470809-74d522a6-dfe9-4106-9740-4d5accd1ea18.png)

This is our EDR log ,for the source side log
![image](https://user-images.githubusercontent.com/32336596/228470884-3ddbaf4b-3aad-4100-baad-0179e06c3621.png)

Sorry I couldn't provide destination side log because it didn't record

**Reply by qoo7972365** [reply]
anyone can help me to resolve this?
